docs/index.rst

```diff
@@ -33,6 +33,6 @@ Starcraft is a member of the Canonical family. It's an open source project
 that warmly welcomes community projects, contributions, suggestions, fixes
 and constructive feedback.
 
-* `Ubuntu Code of Conduct <https://ubuntu.com/community/ethos/code-of-conduct>`_.
-* `Canonical contributor licenses agreement
-  <https://ubuntu.com/legal/contributors>`_.
+* `Ubuntu Code of Conduct <https://ubuntu.com/community/ethos/code-of-conduct>`_
+* `Canonical contributor license agreement
```

Comment:
```suggestion
* `Canonical Contributor License Agreement
```

This is how we name it in `CONTRIBUTING.md`. I _think_ the text on that web page is incorrect. The actual legal documents have title case. But they also contain either "Individual" or "Entity" in the title, so it's all a bit muddy.